docs/pages/plugins-manifest.md

```diff
@@ -81,55 +81,78 @@ Defines the minimum version of Sketch required to run the plugin. This string mu
+| `identifier`  | Defines a unique identifier of the command within the plugin bundle.                                                                                                                                                                                                                                                                                                                                             |
+| `name`        | Provides the name of the command which is used within the _Plugins_ menu.                                                                                                                                                                                                                                                                                                                                        |
+| `shortcut`    | Provides a default keyboard shortcut for the command, e.g. `ctrl shift t`.                                                                                                                                                                                                                                                                                                                                       |
+| `script`      | Specifies the relative path within the plugin bundle's `Sketch` folder to the script implementing the command.                                                                                                                                                                                                                                                                                                   |
+| `handler`     | Specifies the name of function to be called with the command. The function should be declared at the top level of the script, and accepts a single `context` parameter which contains information such as the current document and selection.<br/><br/>If using the `skpm` tool you may `export` the function instead of declaring it, or omit this field entirely and declare the handler with `export default` |
+| `handlers`    | If more fine-grain control is required use the `handlers` field instead of `handler`, see below for in-depth documentation                                                                                                                                                                                                                                                                                       |
 
 ##### `handlers`
 
-An object used to specify the mapping between events (like Actions, calling the command, etc.) and function names.
+An object used to fully specify a command's _setup > run > tearDown_ lifecycle, as well as work with [Actions](/guides/action-api/). When specified the `handler` field is ignored.
```

Comment:
Suggestion for improved formatting:

_setup_ › _run_ › _tearDown_

or 

`setup › run › tearDown`